index.qmd

```diff
@@ -48,7 +47,8 @@ The falcon initiative is an industry collaborative effort under pharmaverse that
 :::
 
 ::: flip-card-back
-Roche
```

Comment:
Do we also want to list the POs or only the developer?

Reply:
Hi @mattkorb, for now I think just developer names and if Vincent wants to add PO names as well he will probably add them within his branch.  Thanks! :)

Reply:
Sounds good 👍